pymeasure/instruments/oxfordinstruments/adapters.py

```diff
@@ -0,0 +1,136 @@
+from pymeasure.adapters import VISAAdapter
+from pyvisa.errors import VisaIOError
+
+import re
+import logging
+
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class RetryVISAError(Exception):
+    pass
+
+
+class RetryingVISAAdapter_oxfordinstruments(VISAAdapter):
```

Comment:
At this moment, this adapter is not used, though it is still required for these instruments; @bilderbuchi, how would you recommend to use such a custom VISAAdapter with the new default connection parameters?